app/views/works/show_multiple.html.erb

```diff
@@ -7,53 +7,60 @@
 <!--subnav-->
 
 <!--main content-->
-<%= form_tag edit_multiple_user_works_path(@user),  :id => 'edit-multiple-works' do %>
-
-  <fieldset>
-    <%= check_all_none %>
+<% if @works.empty? %>
+  <p>
+    <%= t(".no_works") %>
+  </p>
+<% else %>
+  <%= form_tag edit_multiple_user_works_path(@user), id: "edit-multiple-works" do %>
 
     <fieldset>
-      <legend>Actions</legend>
-      <p class="submit">
-        <input type="submit" value="Edit" name="commit">
-        <input type="submit" value="Orphan" name="commit">
-        <input type="submit" value="Delete" name="commit">
-      </p>
-    </fieldset>
+      <%= check_all_none %>
 
-    <ul class="index">
-      <% @works_by_fandom.keys.sort.each do |fandom| %>
-        <li>
-          <fieldset class="fandom listbox">
-            <legend>Select <%= fandom %> works</legend>
-            <h5 class="heading">
-              <%= fandom %>
-              <%= check_all_none %>
-            </h5>
-            <ul class="index concise">
-              <% @works_by_fandom[fandom].each do |work| %>
-                <li>
-                  <label class="action" title="select <%= work.title + (work.posted? ? "" : ts(' (Draft)'))%>"><%= check_box_tag "work_ids[]", work.id, false, id: "work_ids_#{work.id}" %></label>
-                <%= link_to work.title + (work.posted? ? "" : ts(' (Draft)')), work_path(:id => work.id) %>
-		</li>
-              <% end %>
-            </ul>
-          </fieldset>
-        </li>
-      <% end %>
-    </ul>
-  
-    <fieldset>
-      <legend>Actions</legend>
-      <p class="submit">
-        <input type="submit" value="Edit" name="commit">
-        <input type="submit" value="Orphan" name="commit">
-        <input type="submit" value="Delete" name="commit">
-      </p>
-    </fieldset>
+      <fieldset>
+        <legend>Actions</legend>
+        <p class="submit">
+          <input type="submit" value="Edit" name="commit" />
+          <input type="submit" value="Orphan" name="commit" />
+          <input type="submit" value="Delete" name="commit" />
+        </p>
+      </fieldset>
 
-  </fieldset>
+      <ul class="index">
+        <% @works_by_fandom.keys.sort.each do |fandom| %>
+          <li>
+            <fieldset class="fandom listbox">
+              <legend>Select <%= fandom %> works</legend>
+              <h5 class="heading">
+                <%= fandom %>
+                <%= check_all_none %>
+              </h5>
+              <ul class="index concise">
+                <% @works_by_fandom[fandom].each do |work| %>
+                  <li>
+                    <label class="action" title="select <%= work.title + (work.posted? ? "" : t(".draft")) %>">
```

Comment:
I'm checking with our Translation committee to see how they'd like to handle translating this. It might not be helpful to just translate ` (Draft)` here, since the whole title tooltip is `select %{work_title} (Draft)`. I will let you know what they say!

Reply:
It looks like translating ` (Draft)` by itself will be fine! However, since we're showing the work title and status in two places, it would be a little cleaner to set up a variable, like `<% work_title_with_status = work.title + (work.posted? ? "" : t(".draft")) %>`

Then we can have `<label class="action" title="select <%= work_title_with_status %>">` here and `<%= link_to work_title_with_status, work_path(id: work.id) %>` on line 44. 

And when we translate more later, we'll just have to update to something like `t(".select_work", work_title_with_status: work_title_with_status)` for the `title` attribute.